src/cluster/setup.rst

```diff
@@ -163,9 +163,17 @@ then to add nodes by IP address. To get more nodes, go through the same install
 procedure on other machines. Be sure to specify the total number of nodes you
 expect to add to the cluster before adding nodes.
 
-Before you can add nodes to form a cluster, you have to have them
-listen on a public IP address and set up an admin user. Do this, once
-per node:
+Before you can add nodes to form a cluster, you must have them listening on an
+IP address accessible from the other nodes in the cluster.
+Do this once per node:
+
+In file etc/vm.args  to -name couchdb@<this-nodes-ip-address> for each node.
+For clustered setup, each node in the system must have a unique name.
```

Comment:
This paragraph looks like a partial copy of the following one.  Copy and paste error?

Reply:
@raghavtan can you check this comment from @flimzy before I squash and merge? thanks.